mesonbuild/modules/gnome.py

```diff
@@ -990,6 +990,12 @@ def _make_gir_target(
         run_env.set('CFLAGS', [quote_arg(x) for x in env_flags], ' ')
         run_env.merge(kwargs['env'])
 
+        gir_dep, _, _ = self._get_gir_dep(state)
+
+        # response file supported?
+        rspable = (mesonlib.is_windows or mesonlib.is_cygwin()) and \
+            mesonlib.version_compare(gir_dep.get_version(), '>= 1.85.0')
```

Comment:
It's technically supported everywhere, no?

BTW: `mesonlib.is_windows` is true everywhere, it is a function object of type "callable".

Reply:
> It's technically supported everywhere, no?

Yes; however, as when I added response files to glib-mkenums, PATH_MAX issues really affects Windows only.

Reply:
> It's technically supported everywhere, no?

Yes, but Linux people like seeing the whole command-line with `ninja -v`, can avoid another layer of indirection (need to use `-d keeprsp` and then read the rsp file + command-line args) on non-Windows.